lms/templates/ux/reference/instructor_dashboard/cohorts.html

```diff
@@ -0,0 +1,189 @@
+                        <option value="cohort-name-2">Cras mattis consectetur purus sit amet fermentum (8,546)</option>
+                        <option value="cohort-name-3">Donec id elit non mi porta gravida at eget metus. (4)</option>
+                      </select>
+                    </div>
+                  </div>
+                </form>
+
+                <button type="button" class="action-primary action-create"><i class="icon-plus"></i> New Cohort Group</button>
+              </div>
+
+              <!-- Message: Warning -->
+              <div class="msg msg-warning">
+                <h3 class="message-title">You currently have no cohort groups configured</h3>
+                <div class="message-copy">
+                    <p>Please complete your cohort group configuration by creating groups within Studio</p>
+                </div>
+                <div class="message-actions">
+                    <button class="action-primary action-create"><i class="icon icon-plus"></i> Add a Cohort Group</button>
+                </div>
+              </div>
+
+              <!-- Message: Error -->
+              <div class="msg msg-error">
+                <h3 class="message-title">There's currently an error with your cohorts configuration within this course.</h3>
+                <div class="message-copy">
+                    <p>Error output (if any and near-human legible/comprehendable can be displayed here)</p>
+                </div>
+                <div class="message-actions">
+                    <a href="#" class="action-primary action-review">Review Configuration in Studio</a>
+                </div>
+              </div>
+
+              <!-- Managing cohorts -->
+              <section class="cohort-management-settings has-tabs">
+                <h2 class="group-header-title"><span class="title-value">Cohort Name Can be Placed Here and Should Accommodate Almost Everything</span> <span class="group-count">(contains 12,546 students)</span></h2>
+                <div class="cohort-management-group-setup">
+                  <p class="cohort-management-group-text">Students are added to this group only when you provide their email addresses or usernames on this page.<br /><a href="#">What does this mean?</a></p>
+                  <div class="cohort-management-group-actions">
+                    <span class="float-right"><a href="#">Edit settings in Studio</a></a>
+                  </div>
+                </div>
+                <ul class="wrapper-tabs">
+                  <li class="tab"><a href="#">Manage Students in Cohort Groups</a></li>
+                  <li class="tab is-selected"><a href="#">Cohort Group Settings</a></li>
+                </ul>
+
+                <div class="cohort-management-details">
+                  <form action="" method="post" name="" id="cohort-management-details-form" class="cohort-management-details-form">
+                    <h3 class="form-title sr">Cohort Group Details</h3>
```

Comment:
I recommend that we stop using hidden (screen reader only/sr) headings.  I consider this to be a hack.  We should use aria-labels and aria-labelled by when a label is actually needed.  In this case, no label is required.